### PR TITLE
Closes #725. No request for 2013 or later thesis.

### LINKED ISSUE #725: Suppress Request option for any senior thesis deposited after 2012
For those with an embargo display the restrictions note in place of the link to the online thesis in Dspace. Otherwise display the ark pulled in from Dspace.

### PR DESCRIPTION
**Pull request by kevinreiss**

### CODE REVIEW

**Comment by kevinreiss** [comment]
This should fix the mudd thesis problem closes #725.

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -269,7 +272,7 @@ def holding_block_search(document)
           info << content_tag(:span, '', class: 'icon-warning', title: t('blacklight.holdings.paging_request'), 'data-toggle' => 'tooltip').html_safe if pageable?(holding)
         else
           check_availability = false
-          info << content_tag(:span, 'Unavailable', class: 'availability-icon label label-danger', title: 'Availability: On-site', 'data-toggle' => 'tooltip')
+          info << content_tag(:span, 'Unavailable', class: 'availability-icon label label-danger', title: 'Availability: Material Under Embargo', 'data-toggle' => 'tooltip')
```
Comment (tampakis) [inline app/helpers/application_helper.rb]: Can this be sentence-cased to be consistent with the other tooltips?

Comment (kevinreiss) [inline app/helpers/application_helper.rb]: yes, sorry. Should we move these to blacklight.en.yml?

Comment (tampakis) [inline app/helpers/application_helper.rb]: ideally yeah, future work

**Comment by kevinreiss** [comment]
Closes #740, #725, #718, #704, #487, #480, #464.

**Comment by tampakis** [comment]
This is a good improvement. I noticed a couple of issues but they've always been around. Will check to see if there are any tickets open for them.

**Pull request merged**